### PR TITLE
Find min

### PR DESCRIPTION
**Pull request by BooGluten**
A function that finds the minimum value in a list.

### CODE REVIEW

File: findmin.go [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+import("fmt")
+
+func findmin(in []int) int{
+	smallest := in[0]
+    for i := 0; i < len(in); i ++ {
+        var x = in[i]
+
+        if x <= smallest {
+            smallest = x
```
Comment (katkad) [inline findmin.go]: Hi. No need to assign new smallest value, if x  == smallest.

x < smallest
will do

File: findmin.go [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package main
+
+import("fmt")
+
+func findmin(in []int) int{
+	smallest := in[0]
+    for i := 0; i < len(in); i ++ {
+        var x = in[i]
```
Comment (katkad) [inline findmin.go]: Hi. You can start from in[1], because in[0] is already in smallest variable. You do not need to compare in[0] and in[0].

**Review by Mystigan — CHANGES_REQUESTED** [review]
Hi,
Please work on the changes suggested by @katkad . Other than that:
- Please move this file into either `math` or `other` folder`. Rename the package accordingly.
- Remove the main function. Add a test file.

**Comment by tjgurwara99** [comment]
Most comments requesting changes are still valid, even now. Please take a look at our new [contribution guidelines](https://github.com/TheAlgorithms/Go/blob/d2f12849df76203d80762ceca5105531d3424384/CONTRIBUTING.md) and make changes accordingly. Currently this PR is not up to standard. If no changes are made in the next two weeks, I will consider this PR to be stale and will close this PR then.

**Pull request closed without merging**